contracts/tribute/src/contract.rs

```diff
@@ -647,7 +694,34 @@ fn get_top_n_proposals(
 }
 
 #[cfg_attr(not(feature = "library"), entry_point)]
-pub fn migrate(_deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, ContractError> {
+pub fn migrate(deps: DepsMut, _env: Env, msg: MigrateMsg) -> Result<Response, ContractError> {
```

Comment:
wdyt about putting this into a separate "migrate.rs" file? I want to start splitting up the main contract files more, since they tend to get pretty big with our current conventions